### ISSUE TITLE
Android: Subtitle in notification is always filled

### ISSUE THREAD

**Issue by tdekoning** [notifee/react-native-notifee#168]
Every notification from Notifee has a filled `subtitle` property. This results in an added dot after the app name in the notification itself.

Even stranger, when adding a timestamp via the android properties of the notification, it adds 2 dots, like so: 
![image](https://user-images.githubusercontent.com/1109781/95435969-8bf5a800-0953-11eb-9165-3872126caa16.png)

Code to reproduce:
```
await notifee.displayNotification({
    title: 'Hello from my app',
    android: {
      channelId: 'my-channel-id',
    },
  })
```

**Comment by mikehardy** [comment]
I can reproduce this but for some reason had not noticed it until now! Thanks for pointing this out

**Comment by helenaford** [comment]
Fixed in release [v0.15.0](https://notifee.app/react-native/docs/release-notes#0150)
